spec/models/measure_spec.rb

```diff
@@ -95,4 +95,49 @@
       }.from(2).to(1)
     end
   end
+
+  context "notifications" do
+    subject { FactoryBot.create(:measure, notifications: true) }
+    let!(:user) { FactoryBot.create(:user) }
+    let(:user_id) { user.id }
+    let!(:user_measure) { FactoryBot.create(:user_measure, measure: subject) }
+
+    before { allow(::PaperTrail.request).to receive(:whodunnit).and_return(user_id) }
+
+    context "for non 'task' measures" do
+      before { allow(subject).to receive(:task?).and_return(false) }
+
+      it "won't send when relationship_updated_at changes" do
+        expect { subject.touch(:relationship_updated_at) }
+          .not_to change { ActionMailer::Base.deliveries.count }.from(0)
+      end
+    end
+
+    context "for 'task' measures" do
+      before { allow(subject).to receive(:task?).and_return(true) }
+
+      context "when the current user has the user_measure" do
```

Comment:
```suggestion
      context "when the current user owns the task" do
```

Does this make more sense? Basically it's guarding against this case:

> are not the same user updating the database as specified above

Reply:
perfect - thanks!